core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -63,6 +63,19 @@ public static boolean ancestorOf(Table table, long snapshotId, long ancestorSnap
     return ancestorIds(table.currentSnapshot(), table::snapshot);
   }
 
+  /**
+   * Find the oldest Snapshot of a {@link Table}.
```

Comment:
I think this should be more clear, similar to my comments on the docs about the method below. Basically, it should state that the "oldest" is determined by traversing the history of the current table state.

Reply:
fixed it.